Sources/Plasma/PubUtilLib/plPipeline/CMakeLists.txt

```diff
@@ -71,6 +71,8 @@ target_link_libraries(plPipeline
         plSurface
         pfCamera # plCaptureRender, plDynamicEnvMap, plDXPipeline
         pfGameGUIMgr # plCaptureRender
+        freetype
```

Comment:
Please separate 3rd party from the Plasma libraries. Also, please use the imported target `Freetype::Freetype` instead of `freetype`. When CMake sees the namespace delimiter `::`, it will try to resolve the dependency to a target that CMake knows about and will error if it is not found, providing a free configure-time check. Providing a library name (such as `freetype`) will silently pass the equivalent of `-lfreetype` to the linker if no matching CMake target is found, which can potentially result in surprising errors at link time.

Reply:
The namespacing doesn't work. Findfreetype.cmake seems to muck with this.

Reply:
yeah, this is (unfortunately) consistent with what plFontConverter is doing: https://github.com/H-uru/Plasma/blob/fd591cbd88f538fc88ff2eb66a8ebb78f9085022/Sources/Tools/plFontConverter/CMakeLists.txt#L44

Reply:
I had forgotten that upstream freetype exports only `freetype`. CMake's find modules expose the `Freetype::Freetype` target, but we will generally not get that due to loading the freetype cmake config.